jenkins/opensearch/integ-test.jenkinsfile

```diff
@@ -100,7 +100,7 @@ pipeline {
                     env.artifactPath = buildManifestObj.getArtifactRoot(BUILD_JOB_NAME, buildId)
                     env.AGENT_LABEL = agent_nodes["${env.platform}_${architecture}"]
                     env.updateGithubIssues = (params.UPDATE_GITHUB_ISSUES && (env.distribution == 'tar'))
-                    echo "GitHub issue update is set to : ${env.updateGithubIssues}"
+                    echo "GitHub issue update is set to : ${env.updateGithubIssues.toBoolean()}"
```

Comment:
Can we just make env.updateGithubIssues `.toBoolean()` to start with? Instead of adding .toBoolean() to everything?

Reply:
We cannot directly do that. Groovy does not allow method call on variable assignment.